routes/web.php

```diff
@@ -27,6 +27,7 @@
 
 Route::get('/spotify/', 'SpotifyController@index')->name('spotify');
 Route::get('/spotify/track/{id}', 'SpotifyController@trackDetails')->name('spotify.track');
+Route::get('/spotify/history/{date?}', 'SpotifyController@renderDailyHistory')->name('spotify.history');
```

Comment:
Ist das `?` für optional? Hab das noch nie gesehen. :o

Reply:
Yes, it is!